### PR TITLE
add upgrade case OCP-13022

### PR DESCRIPTION
**Pull request by lyman9966**
@jhou1 @sunilcio @weinliu Can you help to take a look ? Thanks
logs is here: 
http://pastebin.test.redhat.com/865326

### CODE REVIEW

File: features/upgrade/node/node-upgrade.feature [hunk 1]
```diff
@@ -0,0 +1,60 @@
+Feature: Node components upgrade tests
+  # @author minmli@redhat.com
+  @upgrade-prepare
+  @admin
+  Scenario: Make sure nodeConfig is not changed after upgrade - prepare
+    Given I switch to cluster admin pseudo user
+    When I run the :label admin command with:
+      | resource      | machineconfigpool         |
+      | name          | master                    |
+      | key_val       | custom-kubelet=small-pods |
```
Comment (liangxia) [inline features/upgrade/node/node-upgrade.feature:10]: Extra white spaces between `resources` and `|`

**Comment by lyman9966** [comment]
@jhou @liangxia I updated the code by fixing format, can you help to have a look?

**Comment by weinliu** [comment]
LGTM

**Pull request merged**